src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -408,6 +452,15 @@
                 "Role": "Server"
             }
         },
+        {
+            "Type": "GitRepoClone",
+            "Parameters": {
+                "Scenario": "CloneSysbenchRepo",
+                "RepoUri": "https://github.com/akopytov/sysbench.git",
+                "PackageName": "sysbench",
```

Comment:
I remember we took this out and use pre-compiled sysbench, why are we going back?

Reply:
ah, i thought it was better to compile it ourselves? otherwise i can just go back to downloading pre-compiled

Reply:
no personally I prefer we compile it if it's not much trouble, but I thought some perf people want it pre-compiled. Let's keep it this way then